### PR TITLE
[hdi] update the replication template

### PR DESCRIPTION
**Pull request by mumian**
### Best Practice Checklist
Check these items before submitting a PR... See the Contribution Guide for the full detail: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/README.md 

1. uri's compatible with all clouds (Stack, China, Government)
1. Staged artifacts use _artifactsLocation & _artifactsLocationSasToken
1. Use resourceGroup().location for resource locations
1. Folder names for artifacts (nestedtemplates, scripts, DSC)
1. Use literal values for apiVersion (no variables)
1. Parameter files (GEN-UNIQUE for value generation and no "changemeplease" values
1. $schema and other uris use https
1. Use uniqueString() whenever possible to generate names for resources.  While this is not required, it's one of the most common failure points in a deployment. 
1. Update the metadata.json with the current date

For details: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/bp-checklist.md

- [ ] - Please check this box once you've submitted the PR if you've read through the Contribution Guide and best practices checklist.

### Changelog

*
*
*

### CODE REVIEW

**Comment by bmoore-msft** [comment]
check your ci failures, need to remove the parameters from the parameter file as well

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 1]
```diff
@@ -215,10 +215,10 @@
       }
     },
     {
-      "apiVersion": "2015-06-15",
-      "name": "[variables('vpnConnections').name1]",
       "type": "Microsoft.Network/connections",
-      "location": "[variables('clusterNode1').location]",
+      "name": "[variables('vpnConnections').name1]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 2]
```diff
@@ -238,12 +238,11 @@
...
       }
     },
-
     {
-      "apiVersion": "2015-06-15",
-      "name": "[variables('vpnConnections').name2]",
       "type": "Microsoft.Network/connections",
-      "location": "[variables('clusterNode2').location]",
+      "name": "[variables('vpnConnections').name2]",
+      "apiVersion": "2015-06-15",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 3]
```diff
@@ -264,202 +263,135 @@
...
+      }
+    },
     {
-      "name": "[variables('clusterNode1').name]",
-      "type": "Microsoft.HDInsight/clusters",
-      "location": "[variables('clusterNode1').location]",
-      "apiVersion": "2015-03-01-preview",
+      "type": "Microsoft.Network/networkInterfaces",
+      "name": "[variables('vNet1').dnsNICName]",
+      "apiVersion": "2016-03-30",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 4]
```diff
@@ -264,202 +263,135 @@
...
+              }
             }
           }
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[variables('vNet1').dnsName]",
+      "apiVersion": "2016-04-30-preview",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 5]
```diff
@@ -264,202 +263,135 @@
...
+      }
+    },
     {
-      "name": "[variables('clusterNode2').name]",
-      "type": "Microsoft.HDInsight/clusters",
-      "location": "[variables('clusterNode2').location]",
-      "apiVersion": "2015-03-01-preview",
+      "type": "Microsoft.Network/networkInterfaces",
+      "name": "[variables('vNet2').dnsNICName]",
+      "apiVersion": "2016-03-30",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json [hunk 6]
```diff
@@ -264,202 +263,135 @@
...
+              }
             }
           }
+        ]
+      }
+    },
+    {
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "[variables('vNet2').dnsName]",
+      "apiVersion": "2016-04-30-preview",
```
Comment (bmoore-msft) [inline 101-hdinsight-hbase-replication-geo/azuredeploy.json]: update apiVersion

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 1 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 2 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 3 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 4 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 5 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 6 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: updated

File: 101-hdinsight-hbase-replication-geo/azuredeploy.json (hunk 1 above)
Comment (mumian) [reply 101-hdinsight-hbase-replication-geo/azuredeploy.json]: Microsoft.HDInsight/clusters hasn't updated the api version for a long time.

**Pull request merged**